src/Evolution/DgSubcell/Tags/Inactive.hpp

```diff
@@ -27,7 +27,8 @@ struct Inactive : db::PrefixTag, db::SimpleTag {
 
 /// \copydoc Inactive
 template <typename TagList>
-struct Inactive<::Tags::Variables<TagList>> : db::PrefixTag, db::SimpleTag {
+struct Inactive<::Tags::Variables<TagList>>
+    : ::Tags::Variables<db::wrap_tags_in<Inactive, TagList>> {
```

Comment:
requesting a change so this doesn't get merged before I have a chance to think about it carefully...